content/reference/user/dedicated_ips/index.md

```diff
@@ -0,0 +1,26 @@
+---
+title: "Dedicated IP's"
+description: "Dedicated IP address for your pools"
+lead: "Dedicated IP address for your pools"
+date: 2021-03-10T09:29:04-05:00
+lastmod: 2021-03-10T09:29:04-05:00
+weight: 10061
+---
+
+## Managing
+
+To access the dedicated IP's section, click on **Dedicated IP's** in the navigation bar.
+
+You can request dedicated IP's through [technical support](https://gatlingcorp.atlassian.net/servicedesk/customer/portal/8/group/12/create/59).
```

Comment:
Perhaps a tip?
And list what we need:
 * organization slug
 * wanted amount of IPs per zone
 * your contact email
 * your GitHub username

A sales person will contact you